libcxx/test/libcxx/utilities/format/format.functions/ascii.pass.cpp

```diff
@@ -8,6 +8,7 @@
 // UNSUPPORTED: c++03, c++11, c++14, c++17
 // UNSUPPORTED: GCC-ALWAYS_INLINE-FIXME
 
+// What would be the best way to force the test with unicode disabled?
 // Force unicode to be disabled.
 // ADDITIONAL_COMPILE_FLAGS: -D_LIBCPP_HAS_NO_UNICODE
```

Comment:
The approach by forcing the flags does not work after this patch; it probably should have never worked.
What would be the best approach, only define the macro when not defined yet, or mark this with
`REQUIRES: libcpp-has-no-unicode` ?

The same for the next file.